library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
```

Comment:
If I did this I'd name this structure "rectangle" not "window" and create two functions: make_rect(x, y, w, h) and draw_outline(rectangle). For some reason the name "window" confuses me in this case.